dotnet/src/IntegrationTests/Connectors/AzureOpenAI/AzureOpenAIChatCompletionFunctionCallingTests.cs

```diff
@@ -81,7 +81,19 @@ public async Task CanAutoInvokeKernelFunctionsStreamingAsync()
     public async Task CanAutoInvokeKernelFunctionsWithComplexTypeParametersAsync()
     {
         // Arrange
-        var kernel = this.CreateAndInitializeKernel(importHelperPlugin: true);
+        var kernel = this.CreateAndInitializeKernel();
+        kernel.ImportPluginFromFunctions("HelperFunctions",
+        [
+            kernel.CreateFunctionFromMethod((WeatherParameters parameters) =>
+            {
+                if (parameters.City.Name == "Dublin" && (parameters.City.Country == "Ireland" || parameters.City.Country == "IE"))
```

Comment:
Can we make those case insensitive?

```suggestion
                if (parameters.City.Name == "Dublin" && (parameters.City.Country == "Ireland" || parameters.City.Country == "IE"))
```